diff -up plasma-desktop-master/applets/kicker/package/contents/ui/ActionMenu.qml.omv~ plasma-desktop-master/applets/kicker/package/contents/ui/ActionMenu.qml
diff -up plasma-desktop-master/applets/kickoff/package/contents/ui/AbstractKickoffItemDelegate.qml.omv~ plasma-desktop-master/applets/kickoff/package/contents/ui/AbstractKickoffItemDelegate.qml
--- plasma-desktop-master/applets/kickoff/package/contents/ui/AbstractKickoffItemDelegate.qml.omv~	2023-07-26 02:20:51.995452349 +0200
+++ plasma-desktop-master/applets/kickoff/package/contents/ui/AbstractKickoffItemDelegate.qml	2023-07-26 02:22:05.532941772 +0200
@@ -164,7 +164,7 @@ T.ItemDelegate {
             // built into QQuickListView::setCurrentIndex() already
             root.view.currentIndex = index
         }
-        onPressed: {
+        onPressed: (mouse) => {
             // Select and focus on press to improve responsiveness and touch feedback
             view.currentIndex = index
             root.forceActiveFocus(Qt.MouseFocusReason)
@@ -184,13 +184,17 @@ T.ItemDelegate {
                 })
             }
         }
-        onClicked: if (mouse.button === Qt.LeftButton) {
-            root.action.trigger()
+        onClicked: (mouse) => {
+            if (mouse.button === Qt.LeftButton) {
+                root.action.trigger()
+            }
         }
         // MouseEvents for pressAndHold use Qt.MouseEventSynthesizedByQt for mouse.source,
         // which makes checking mouse.source for whether or not touch input is used useless.
-        onPressAndHold: if (mouse.button === Qt.LeftButton) {
-            root.openActionMenu(mouseX, mouseY)
+        onPressAndHold: (mouse) => {
+            if (mouse.button === Qt.LeftButton) {
+                root.openActionMenu(mouseX, mouseY)
+            }
         }
     }
 
diff -up plasma-desktop-master/applets/kickoff/package/contents/ui/ActionMenu.qml.omv~ plasma-desktop-master/applets/kickoff/package/contents/ui/ActionMenu.qml
--- plasma-desktop-master/applets/kickoff/package/contents/ui/ActionMenu.qml.omv~	2023-07-26 02:17:48.115663920 +0200
+++ plasma-desktop-master/applets/kickoff/package/contents/ui/ActionMenu.qml	2023-07-26 02:19:20.199233715 +0200
@@ -38,8 +38,8 @@ Item {
         active: root.actionList !== null
         model: root.actionList
         delegate: menuItemComponent
-        onObjectAdded: menu.addMenuItem(object)
-        onObjectRemoved: menu.removeMenuItem(object)
+        onObjectAdded: (index, object) => { menu.addMenuItem(object) }
+        onObjectRemoved: (index, object) => { menu.removeMenuItem(object) }
     }
 
     Component {
