diff -up plasma-desktop-master/layout-templates/org.kde.plasma.desktop.defaultPanel/contents/layout.js.omv~ plasma-desktop-master/layout-templates/org.kde.plasma.desktop.defaultPanel/contents/layout.js
--- plasma-desktop-master/layout-templates/org.kde.plasma.desktop.defaultPanel/contents/layout.js.omv~	2023-08-03 16:00:45.848875422 +0200
+++ plasma-desktop-master/layout-templates/org.kde.plasma.desktop.defaultPanel/contents/layout.js	2023-08-03 16:02:21.341789290 +0200
@@ -21,15 +21,6 @@ if (panel.formFactor === "horizontal") {
     }
 }
 
-var kickoff = panel.addWidget("org.kde.plasma.kicker")
-kickoff.currentConfigGroup = ["Shortcuts"]
-kickoff.writeConfig("global", "Alt+F1")
-
-//panel.addWidget("org.kde.plasma.showActivityManager")
-panel.addWidget("org.kde.plasma.pager")
-panel.addWidget("org.kde.plasma.icontasks")
-panel.addWidget("org.kde.plasma.marginsseparator")
-
 /* Next up is determining whether to add the Input Method Panel
  * widget to the panel or not. This is done based on whether
  * the system locale's language id is a member of the following
@@ -69,11 +60,20 @@ var langIds = ["as",    // Assamese
                "zh_CN", // Simplified Chinese
                "zh_TW"] // Traditional Chinese
 
 if (langIds.indexOf(languageId) != -1) {
     panel.addWidget("org.kde.plasma.kimpanel");
 }
 
-panel.addWidget("org.kde.plasma.systemtray")
-panel.addWidget("org.kde.plasma.digitalclock")
-panel.addWidget("org.kde.plasma.showdesktop")
+panel.addWidget("org.kde.plasma.systemtray")
+
+panel.addWidget("org.kde.plasma.marginsseparator")
+panel.addWidget("org.kde.plasma.icontasks")
+panel.addWidget("org.kde.plasma.pager")
+//panel.addWidget("org.kde.plasma.showActivityManager")
+panel.addWidget("org.kde.plasma.digitalclock")
+panel.addWidget("org.kde.plasma.showdesktop")
+
 
+var kickoff = panel.addWidget("org.kde.plasma.kicker")
+kickoff.currentConfigGroup = ["Shortcuts"]
+kickoff.writeConfig("global", "Alt+F1")
