--- plasma-desktop-5.10.1/applets/kimpanel/backend/scim/main.cpp.omv~	2017-06-12 12:34:27.312831317 +0200
+++ plasma-desktop-5.10.1/applets/kimpanel/backend/scim/main.cpp	2017-06-12 12:34:47.465364727 +0200
@@ -27,6 +27,8 @@
 
 #define QT_NO_KEYWORDS
 
+#define _GLIBCXX_USE_CXX11_ABI 0 // for some reason scim libs use the old ABI
+
 #include <config-scim.h>
 #include <sys/wait.h>
 #include <sys/types.h>
