diff -up plasma-desktop-master/appiumtests/CMakeLists.txt.omv~ plasma-desktop-master/appiumtests/CMakeLists.txt
diff -up plasma-desktop-master/appiumtests/inputsynth/CMakeLists.txt.omv~ plasma-desktop-master/appiumtests/inputsynth/CMakeLists.txt
--- plasma-desktop-master/appiumtests/inputsynth/CMakeLists.txt.omv~	2023-10-14 21:20:41.087834740 +0200
+++ plasma-desktop-master/appiumtests/inputsynth/CMakeLists.txt	2023-10-14 21:21:22.566202117 +0200
@@ -26,6 +26,6 @@ target_link_libraries(inputsynth
 # Do not really install the library
 if(DEFINED ENV{KDECI_BUILD})
 install(TARGETS inputsynth ${KDE_INSTALL_TARGETS_DEFAULT_ARGS})
-else()
-install(TARGETS inputsynth DESTINATION ${CMAKE_BINARY_DIR})
+#else()
+#install(TARGETS inputsynth DESTINATION ${CMAKE_BINARY_DIR})
 endif()
