diff -up plasma-desktop-6.1.0/layout-templates/org.kde.plasma.desktop.defaultPanel/contents/layout.js.1~ plasma-desktop-6.1.0/layout-templates/org.kde.plasma.desktop.defaultPanel/contents/layout.js
--- plasma-desktop-6.1.0/layout-templates/org.kde.plasma.desktop.defaultPanel/contents/layout.js.1~	2024-06-18 22:40:50.821569964 +0200
+++ plasma-desktop-6.1.0/layout-templates/org.kde.plasma.desktop.defaultPanel/contents/layout.js	2024-06-18 22:46:04.613056095 +0200
@@ -21,7 +21,7 @@ if (panel.formFactor === "horizontal") {
     }
 }
 
-panel.addWidget("org.kde.plasma.kickoff")
+panel.addWidget("org.kde.plasma.kicker")
 //panel.addWidget("org.kde.plasma.showActivityManager")
 panel.addWidget("org.kde.plasma.pager")
 panel.addWidget("org.kde.plasma.icontasks")
